pegasus/rake/db.rake

```diff
@@ -56,11 +76,4 @@ namespace :db do
     Sequel::Migrator.run(DB, migrations_dir)
     Rake::Task['db:version'].execute
   end
-
-  task :help do
-    puts 'db:version'
-    puts 'db:migrate'
-    puts 'db:rollback'
-    puts 'db:reset'
-  end
```

Comment:
I wasted too much time looking for a way to invoke `rake --tasks` from inside the `:default` or `:help` tasks; in the end I figured we'll mostly expect people to learn the convention.